### PR TITLE
NER versioning updates and fix for no domain for collections

### PR DESCRIPTION
**Pull request by anneCarlson**
Reviewers: 
----------

- [x] @madisonmay 
- [ ] @anneCarlson

What's the purpose of the PR?
-----------------------------

Updates version number of ner.
https://trello.com/c/voWeqH80/1986-3-ner-updated-in-client-libs-default-version
Fix for custom domain is in there as well.

Description of changes
----------------------

Notes to reviewers
------------------

Keep in mind while reviewing code:
----------------------------------
- Is relevant code tested?
- Are added functions/methods documented?
- Separation of concerns (SOC)
- Don't repeat yourself (DRY) 
- Limit the number of positional arguments
- Is function/method length reasonable?
- Can code be broken down into smaller components?
- Where do added functions/methods belong?
- Are variable names descriptive?
- Are errors handled appropriately?
- Can logic be simplified?
- Does the code make sense in context? (expand the diff)

### CODE REVIEW

File: .Rhistory [hunk 1]
```diff
@@ -18,3 +18,5 @@ quit()
 source("http://bioconductor.org/biocLite.R")
 biocLite("EBImage")
 quit()
+devtools::test()
+sublime .
```
Comment (madisonmay) [inline .Rhistory]: ?

File: R/api_handler.R [hunk 2]
```diff
@@ -39,17 +39,31 @@ make_request <- function(data, api, version = NULL, apis = NULL, method = NULL,
   body <- toJSON(kwargs)
 
   response <- POST(url, accept_json(), headers, body = body)
+  print('##################################')
```
Comment (sihrc) [inline R/api_handler.R]: print statements.

File: R/collection.R [hunk 3]
```diff
@@ -50,8 +50,11 @@ setGeneric(name="addData",
 #' addData(collection, test_data)
 setMethod(f="addData",
           signature="Collection",
-          definition=function(collection_object, data, version = NULL, domain = NULL, ...) {
-                collection_object@domain <- ifelse(domain!=NULL, domain, collection_object@domain)
+          definition=function(collection_object, data, version = NULL, domain=NULL, ...) {
+                if (! is.null(domain)) {
```
Comment (sihrc) [inline R/collection.R]: pretty sure, this doesn't change anything. I don't think this conditional is necessary.

File: R/collection.R [hunk 4]
```diff
@@ -62,7 +65,11 @@ setMethod(f="addData",
                 } else {
                     data[1] = format_image(data[[1]], 48)
                 }
-                make_request(data, 'custom', version=version, collection = collection_object@name, method = "add_data", domain=collection_object@domain, ...)
+                if ( is.null(collection_object@domain) )  {
+                    make_request(data, 'custom', version=version, collection = collection_object@name, method = "add_data", ...)
+                } else{
+                    make_request(data, 'custom', version=version, collection = collection_object@name, method = "add_data", domain=collection_object@domain, ...)
+                }
```
Comment (sihrc) [inline R/collection.R]: Instead of a change like this, we should simply exclude NULL values inside make_request.

**Comment by sihrc** [comment]
version 1 tests?

File: R/collection.R (hunk 3 above)
Comment (madisonmay) [inline R/collection.R]: This won't set domain unless it's not null, right?

File: R/collection.R (hunk 4 above)
Comment (madisonmay) [inline R/collection.R]: Agreed.

File: R/collection.R (hunk 3 above)
Comment (anneCarlson) [inline R/collection.R]: This errored if both were NULL, strangely.

**Pull request merged**